src/index.js

```diff
@@ -0,0 +1,68 @@
+// Libraries
+import express from 'express';
+import { ApolloServer } from 'apollo-server-express';
+import cors from 'cors';
+import cookieParser from 'cookie-parser';
+// import csurf from 'csurf';
+
+// GraphQL Schema
+import schema from './graphql/index.js';
+
+// Initialize Firebase and Mongoose Admin SDK
+import './config/mongoose.js';
+import './config/firebase.js';
+
+// Utilities
+import {
+  consoleSuccess,
+  consoleError,
+  consoleInfo,
+  consoleWarn,
+  consoleLog,
+} from './utils/console.js';
+
+// Create Express app instance
+const app = express();
+
+// Setup Cross-Origin Resource Sharing for the development environment
+// localhost:3000 would be the frontend port on which the app is running
+var corsOptions = {
+  origin:
+    process.env.NODE_ENV !== 'production'
+      ? 'http://localhost:3000'
```

Comment:
line 55 says port 8000, here is 5000

Reply:
ya, the server will start on port 8000 and port 3000 will be the frontend. So in development, we want redirects from this site, which is why this.

Reply:
aah, I see. makes sense

Reply:
small confusion, all sorted